### ISSUE TITLE
Upgrade dependencies

### ISSUE THREAD

**Issue by xPaw** [sysgears/mochapack#47]
We get following warnings in our project:
```
warning mochapack > babel-runtime > core-js@2.6.10: core-js@<3.0 is no longer maintained and not recommended for usage due to the number of issues. Please, upgrade your dependencies to the actual version of core-js@3.
warning mochapack > chokidar > fsevents@1.2.9: One of your dependencies needs to upgrade to fsevents v2: 1) Proper nodejs v10+ support 2) No more fetching binaries from AWS, smaller package size
```

**Comment by larixer** [comment]
Yeah, it would be great to upgrade these dependencies, though it might be not a very easy journey. Babel 6 -> Babel 7 upgrade is needed, it should be generally straightforward. But I'm not sure that all the latest deps (for example watchpack) can actually use the latest fsevents version. And I'm not also sure if multiple chokidar/fsevents versions can coexist peacefully, I saw at least once the multiple chokidar versions do not work well together:
https://github.com/sysgears/mochapack/issues/39

**Comment by xPaw** [comment]
Have you considered just using native `fs.watch`?

Looking at watchpack, they switched to it in v2 beta releases: https://github.com/webpack/watchpack/commit/805b66502e43be4b1892d2cc0b160e4c53e32e38#diff-b9cfc7f2cdf78a7f4b91a753d10865a2

**Comment by larixer** [comment]
@xPaw Well, `chokidar` also uses native `fs.watch`:
https://github.com/paulmillr/chokidar#how
It just wraps it. And `watchpack` has also non-trivial wrapping code around native `fs.watch`. I am basically okay with either option: native `fs.watch` or library around native `fs.watch`.

**Comment by xPaw** [comment]
Well, chokidar comes with the baggage of fsevents, which has a native package (and all of its problems)

**Comment by specialistvlad** [comment]
Please update chokidar
